next/components/forms/widget-components/InputField/InputField.tsx

```diff
@@ -156,7 +156,7 @@ const InputField = forwardRef<HTMLInputElement, InputFieldProps>(
               {leftIconSwitcher(leftIcon)}
             </span>
           )}
-          <input {...inputProps} ref={ref} name={inputProps.id} className={style} />
+          <input {...inputProps} ref={ref} name={inputProps.id} className={style} data-cy={`input-${inputProps.name}`}/>
```

Comment:
do these make sense when they are dynamic ? I understand they help you target directly with just this, put since you're relying on the non-cypress world for the naming, why not select using `data-cy` together with the value of `name` param for example ?

same question elsewhere on the form components